apps/templates/content_templates.py

```diff
@@ -335,7 +352,8 @@ def get_item_from_template(template):
     """
     item = template.get('data', {})
     item[ITEM_STATE] = CONTENT_STATE.SUBMITTED
-    item['task'] = {'desk': template.get('template_desk'), 'stage': template.get('template_stage')}
+    if len(template.get('template_desks', [])) == 1:
```

Comment:
if multiple desks are assigned to a template and then if the template is used to schedule content creation then the item won't be located on any desk. We don't have a user assigned `task.user` for the scheduled content and `SUBMITTED` state is only allowed on desk. imo, if the item is created using scheduling then it would be hard to locate this item.